tools/scoring/tests/dimensions/raw_field_selection_test.py

```diff
@@ -19,35 +19,129 @@
 from score.dimensions.raw_field_selection import RawFieldSelection
 from score.constants import FileTypes, DimensionCategories
 
+from tests.helper import TestHelper
+
 PROPOSED, SOLUTION = FileTypes
 
 
 class RawFieldSelectionTest(absltest.TestCase):
   def setUp(self):
     super().setUp()
-    # TODO: add real data (append cases to existing tests)
-    translations = {'cloud_device_id': {PROPOSED: [], SOLUTION: []}}
-    self.raw_field_selection = RawFieldSelection(
-        translations=translations).evaluate()
+    one_entity_file_path = (
+        'tests/samples/proposed/raw_field_selection_one_entity.yaml')
```

Comment:
can we reuse existing files? they seem the same?

Reply:
They have the same content in this circumstance but I maintain that they should be separate for a few reasons:
- The purpose of unit tests is isolation, and combining them breaks that pattern
- Combining them, especially when the content does not overlap, makes it very difficult to reason about what is being tested and hampers maintainability
- Given that some tests require their own files, discerning the reasoning behind the inconsistency will be challenging down the line

What advantage do you think combining them would provide?

Reply:
repeating unnecessary content?

Reply:
Updated the files to differentiate them.